pkg/xray/resource/policies_test.go

```
@@ -0,0 +1,59 @@
+	for _, caller := range []string{"Create", "Update"} {
+		t.Run(caller, func(t *testing.T) {
+			var sawRequest bool
+			var gotContentType string
+
+			// Stands in for the proxy/load balancer in front of Xray: it only
+			// records what the outgoing GET actually carried.
+			proxy := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+				sawRequest = true
+				gotContentType = r.Header.Get("Content-Type")
+				w.Header().Set("Content-Type", "application/json")
+				w.Write([]byte(`{}`))
```

golangci-lint (2.13.2)

[error] Error return value of `w.Write` is not checked

(errcheck)